phpunit.xml.dist

```diff
@@ -17,7 +17,7 @@
     <php>
         <ini name="error_reporting" value="-1"/>
         <ini name="memory_limit" value="-1" />
-        <server name="DB_URL" value="mysql://pimcore:pimcore@db:3306/pimcore_test?serverVersion=mariadb-10.5.13"/>
+        <server name="DB_URL" value="mysql://pimcore:pimcore@db:3306/pimcore_test?serverVersion=mariadb-10.10.2"/>
```

Comment:
I would define this in the `docker-compose.yaml`, as it may be different depending on the context (e.g. when you want to run the tests in a GitHub action).

Reply:
This would be my next step :)

Reply:
Done